state/storage.go

```diff
@@ -1194,7 +1194,7 @@ func (sb *storageBackend) DestroyUnitStorageAttachments(unit names.UnitTag) (err
 
 // DetachStorage ensures that the storage attachment will be
 // removed at some point.
-func (sb *storageBackend) DetachStorage(storage names.StorageTag, unit names.UnitTag, force bool) (err error) {
+func (sb *storageBackend) DetachStorage(storage names.StorageTag, unit names.UnitTag, force bool, maxWait time.Duration) (err error) {
```

Comment:
maxWait isn't used here yet? If that's the case we should hide it from the CLI - we don't want to have visible options that don't do anything. Although it's good to have the facade bump in already.

Reply:
Well, max wait is only used in scheduled jobs, none of which exist for storage yet. I'll hide the option in CLI but will keep it everywhere else.